Src/WitsmlExplorer.Frontend/components/Modals/CopyMnemonicsModal.tsx

```diff
@@ -0,0 +1,231 @@
+
+    setIsLoading(false);
+  };
+
+  async function orderPasteJob() {
+    const jobType =
+      startIndex !== undefined ? JobType.CopyLogData : JobType.CopyComponents;
+
+    const copyJob: CopyComponentsJob = {
+      source: sourceReferences,
+      target: targetReference,
+      startIndex: startIndex,
+      endIndex: endIndex
+    };
+
+    if (sourceServer) {
+      AuthorizationService.setSourceServer(sourceServer);
+      JobService.orderJobAtServer(jobType, copyJob, targetServer, sourceServer);
+    } else {
+      await JobService.orderJob(jobType, copyJob);
+    }
+  }
+
+  async function orderDeleteInsertJob() {
+    const parentUid = targetReference.uid;
+    const parentType = getParentType(ComponentType.Mnemonic);
+
+    const targetParent = await ObjectService.getObject(
+      targetReference.wellUid,
+      targetReference.wellboreUid,
+      parentUid,
+      parentType,
+      undefined,
+      targetServer
+    );
+
+    const allTargetComponents = await ComponentService.getComponents(
+      targetReference.wellUid,
+      targetReference.wellboreUid,
+      targetReference.uid,
+      ComponentType.Mnemonic,
+      targetServer
+    );
+
+    const indexCurve = (targetParent as LogObject)?.indexCurve;
+
+    const targetComponentsToDelete = allTargetComponents.filter(
+      (c) => c.mnemonic !== indexCurve
+    );
+
+    if (targetComponentsToDelete.length == 0) {
+      await orderPasteJob();
+    } else {
+      const deleteJob: DeleteComponentsJob = {
+        toDelete: createComponentReferences(
+          targetComponentsToDelete.map((component) => component.mnemonic),
+          targetParent,
+          ComponentType.Mnemonic,
+          targetServer.url
+        )
+      };
+
+      const copyJob: CopyComponentsJob = {
+        source: sourceReferences,
+        target: targetReference
+      };
+
+      const replaceJob: ReplaceComponentsJob = { deleteJob, copyJob };
+
+      if (sourceServer) {
+        await JobService.orderJobAtServer(
+          JobType.ReplaceComponents,
+          replaceJob,
+          targetServer,
+          sourceServer
+        );
+      } else {
+        await JobService.orderJob(JobType.ReplaceComponents, replaceJob);
+      }
+    }
+  }
+
+  return (
+    <>
+      <ModalDialog
+        heading={`Paste`}
+        confirmText={`Submit`}
+        content={
+          <ContentLayout>
+            <TextLayout>Choose paste option:</TextLayout>
+            <RadioLayout>
+              <RadioItemLayout>
+                <div>
+                  <Radio
+                    checked={
+                      selectedCopyMnemonicsType ===
+                      CopyMnemonicsType.DeleteInsert
+                    }
+                    onChange={() =>
+                      setCopyMnemonicsType(CopyMnemonicsType.DeleteInsert)
+                    }
+                  />
+                </div>
+                <div>
+                  <div>Delete/Insert</div>
+                  <div>
```

Comment:
Can you use EDS' Typography for text instead of div? It's what we mostly use in WEx.
Link to EDS if you want to see the docs: https://storybook.eds.equinor.com/?path=/docs/introduction--docs